### ISSUE TITLE
Not displaying characters with tones in systemd boot menu properly

### ISSUE THREAD

**Issue by peigongdsd** [nix-community/lanzaboote#344]
The latest version of nixos is named 24.11.20240526.a457dae (Vicuña), while the character ñ of  Vicuña is not properly displaying in systemd-boot shipped with lanzaboote. See the latest entry in the following image.

![image](https://github.com/nix-community/lanzaboote/assets/51317171/85839100-2835-431b-9259-3cbe7df5a688)

**Comment by nikstur** [comment]
I don't believe this is a lanzaboote issue. On my system, the character is displayed correclty. It suspect this might be an issue in your UEFI implementation (or less likely a bug in systemd-boot).

**Comment by RaitoBezarius** [comment]
I confirmed this works out of the box on my:

> UEFI 2.80 (American Megatrends 5.26)

system.

i.e. the fonts are available on modern up-to-date UEFI.

**Comment by RaitoBezarius** [comment]
@nikstur We could consider implementing a "my firmware is bad" mode to strip accents and so on, to be honest.

**Comment by peigongdsd** [comment]
So I think this is not the right place to ask help for this problem. Maybe I should turn to systemd-boot?

**Comment by RaitoBezarius** [comment]
systemd-boot is not responsible for this, neither. This is an OEM problem with your firmware. You should contact your PC manufacturer and ask them to push a firmware update that includes a font with those glyphs.

**Comment by nikstur** [comment]
> We could consider implementing a "my firmware is bad" mode

The problem with these workarounds is that there's endless things we could address with hardly any ability to test it.

**Comment by RaitoBezarius** [comment]
Well, here, the test is pretty trivial, you take EDK2, remove the font that contains a certain glyph, boot and observe garbled output, the test can confirm that we ASCII-ify things or whatever.

**Comment by nikstur** [comment]
Likely an upstream issue. Anyways, there is nothing we can do.
